src/renderer/chat/ChatTab.ts

```diff
@@ -246,6 +247,17 @@ export default class ChatTab {
       if (this.textAreaElt.value.length > 0) return;
       this.textAreaElt.value = this.previousMessage;
       event.preventDefault();
+    }else if(event.keyCode == 9 /* TAB */){        
+        let toComplete = "";
+        for(let i = this.textAreaElt.selectionStart - 1; i >= 0; --i){
+            if(this.textAreaElt.value.charAt(i) == " ") break;
+            toComplete = this.textAreaElt.value.charAt(i) + toComplete;
+        }
+        if(toComplete !== "")
+            for(let i = 0; i < this.users.length; ++i)
+                if(this.users[i].indexOf(toComplete) == 0)
+                    this.textAreaElt.value = this.textAreaElt.value.replace(toComplete, this.users[i]);

```

Comment:
This might replace an earlier instance of the word to complete. Fixed in b7270e85e018b5030d174d84252b780c024166a8.